natlas-server/Pipfile

```diff
@@ -32,6 +28,10 @@ webpack-manifest = "~=2.1.1"
 natlas-libnmap = "*"
 cyclicprng = "*"
 mysqlclient = "*"
+opentelemetry-sdk = "*"
```

Comment:
Can you add ~= version markers to these packages? You don't need to go all the way to patch version, just `~=1.1` for example would be fine.

We generally don't want `*` markers because it becomes much more likely to have unexpected package updates that break the system. Even with the lock file, anytime someone installs a new package, it'll update all existing ones, so we want to try to make sure they are reasonably safe to do so.

Not everyone uses semver, so the `~=` syntax doesn't necessarily work everywhere, but it's a good pattern to get into.